android-components/components/browser/state/src/main/java/mozilla/components/browser/state/state/SearchState.kt

```diff
@@ -38,6 +38,7 @@ data class SearchState(
     val additionalSearchEngines: List<SearchEngine> = emptyList(),
     val additionalAvailableSearchEngines: List<SearchEngine> = emptyList(),
     val hiddenSearchEngines: List<SearchEngine> = emptyList(),
+    val disabledSearchEngineShortcutsIds: List<String> = emptyList(),
```

Comment:
nit: Let's remove "shortcuts" from the naming. Shortcuts means something different in both of our supported browsers and they are also an application-level concept that does not need to be here.
```suggestion
    val disabledSearchEngineIds: List<String> = emptyList(),
```

Reply:
implemented